qemu/tests/cfg/ctrl_vlan.cfg

```diff
@@ -0,0 +1,19 @@
+- ctrl_vlan: install setup image_copy unattended_install.cdrom
+    type = ctrl_vlan
+    image_snapshot = yes
```

Comment:
Just a quick question, why need snapshot here?

Reply:
To keep image clean after specific setting configure on guest.